integration_tests/integration/court-register.spec.ts

```diff
@@ -179,6 +180,28 @@ context('Court register', () => {
     CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court').markAsOpenButton().click()
     CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court').activatedConfirmationBlock().should('exist')
   })
+  describe('amending a court', () => {
+    beforeEach(() => {
+      IndexPage.verifyOnPage().courtRegisterLink().click()
+      AllCourtsPage.verifyOnPage().viewCourtLink('SHFMC').should('contain.text', 'Sheffield Magistrates Court').click()
+    })
+    it('should show summary of court with link to amend', () => {
+      const courtDetailsPage = CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court')
+      courtDetailsPage.courtDetailsSection().should('contain.text', 'Sheffield Magistrates Court')
```

Comment:
will be tested in line below anyway

Reply:
Yeah, I need to amend the test data so these two fields are different enough